### ISSUE TITLE
Can't edit things inside Dirac brakets

### ISSUE THREAD

**Issue by AllanChain** [arnog/mathlive#2138]
## Description

I can't edit things inside brakets. The cursor can't go inside.

### Steps to Reproduce

1. Go to the MathLive demo page https://cortexjs.io/mathlive/demo/
2. Put `\ket{\psi}` inside the LaTeX box
3. Try to edit inside $\ket{}$

### Actual Behavior

The cursor can't go inside by clicking or arrow keys.

https://github.com/arnog/mathlive/assets/36528777/c62a608b-beaa-4d18-bb4d-a42beab7d286

### Expected Behavior

When clicking inside the braket, the cursor goes inside and I can edit, just like the `\lbrack...\rbrack` case.

### Environment

**MathLive version** 0.95.5

**Operating System** Linux

**Browser** Firefox and Chrome

**Comment by arnog** [comment]
Depends on #1565

**Comment by AllanChain** [comment]
I don't think it is an exact dupe of "editing macros with arguments", because this issue consists of two parts:

1. Can't edit `\ket{\placeholder}` because it's defined with in `\mathinner`
2. For `\Ket{\placeholder}` which doesn't have `\mathinner`, I can edit but same issue as #1565

Even if #1565 is resolved, this issue still has the first part unresolved.

Maybe I can open a new issue to track the `\mathinner` part?

**Comment by arnog** [comment]
You're right. I've checked in a fix for the inner issue.
